include/net/cloud.h

```diff
@@ -291,9 +270,8 @@ static inline int cloud_input(const struct cloud_backend *const backend)
 static inline int cloud_user_data_set(struct cloud_backend *const backend,
 				      void *user_data)
 {
-	if (backend == NULL
-	    || backend->api == NULL
-	    || backend->api->user_data_set == NULL) {
+	if (backend == NULL || backend->api == NULL ||
+	    backend->api->user_data_set == NULL) {
```

Comment:
Is this uncrustify or clang-format? Looks pretty good

Reply:
clang-format